src/EntryPoints/Specials/SpecialPendingApprovals.php

```diff
@@ -84,7 +84,7 @@ private function createPendingApprovalRow( PendingApproval $pendingApproval ): s
 			"\n",
 			[
 				Html::rawElement( 'td', [], $this->linkRenderer->makeLink( $pendingApproval->title ) ),
-				Html::element( 'td', [], implode( ', ', $this->getCategoryTitlesFromDbKeys( $pendingApproval->categories ) ) ),
+				Html::element( 'td', [], implode( ', ', $pendingApproval->categories ) ),
```

Comment:
PendingApproval object contains the human-readable name, so no longer necessary to convert here.